ci/appveyor/test.bat

```diff
@@ -0,0 +1,43 @@
+@echo on
+@rem Matrix-driven Appveyor CI test script
+@rem Currently only supports MSYS2 builds.
+@rem See https://www.appveyor.com/docs/installed-software#mingw-msys-cygwin
+@rem Required vars:
+@rem    TEST_STEP
+@rem    WIN_MSYS2_ROOT
+
+SETLOCAL ENABLEEXTENSIONS
+cd %APPVEYOR_BUILD_FOLDER%
+
+@echo Current time: %time%
+rem Set the paths appropriately
+
+call "C:\Program Files (x86)\Microsoft Visual Studio %VS_VERSION%\VC\vcvarsall.bat" %PLATFORM%
+@echo on
+SET PATH=%WIN_MSYS2_ROOT%\mingw64\bin;%WIN_MSYS2_ROOT%\usr\bin;%PATH%
+
+GOTO TEST_%TEST_STEP%
+
+:TEST_ct
+@echo Current time: %time%
+rem Run test: ct
+bash -lc "cd %BUILD_PATH% && epmd -daemon && make ct"
```

Comment:
Any reasons not to call the Makefile?

Reply:
What do you mean? Make is called at the end.

Reply:
🤦‍♂️ please ignore.